Documentation/HowToUseWCFInContainers.md

```diff
@@ -0,0 +1,142 @@
+This document describes how to get started with WCF for .NET Core in containers.
+
+### 1. Sample app
+These are the steps for creating a sample WCF service and client. If you already have a sample WCF service app (or even host it in a container like described in an excellent Jeffrey�s blog post https://blogs.msdn.microsoft.com/webdev/2017/02/20/lets-try-wcf-self-hosted-services-in-a-container/), and a corresponding .NET Core WCF client project then you may skip the sample app creation and proceed to step 2. 
```

Comment:
replace "in an excellent Jeffrey’s blog post https://..."
with "in an excellent blog post by Jeffrey T. Fritz at https://..."